vita3k/gui/src/game_selector.cpp

```diff
@@ -178,10 +192,17 @@ void draw_game_selector(GuiState &gui, HostState &host) {
                     }
                     if (ImGui::IsMouseClicked(0))
                         selected[0] = true;
+                    if (gui.title_id != game.title_id)
+                        gui.title_id = game.title_id;
                 }
+                ImGui::OpenPopupOnItemClick("#game_context_menu", 1);
             }
             ImGui::NextColumn();
             ImGui::Selectable(game.title_id.c_str(), &selected[1], ImGuiSelectableFlags_SpanAllColumns, ImVec2(0, icon_size));
+            if (ImGui::IsItemHovered() && gui.title_id != game.title_id)
```

Comment:
```suggestion
            if (ImGui::IsItemHovered())
```
no

Reply:
For me is yes haha, i don't want change value if not needed is more quickely skip compared set value

Reply:
XD I can't handle it at all.

Reply:
Not worry, is not important, is just my logical wanted perfectioniste haha